styles.css

```diff
@@ -0,0 +1,100 @@
+/* Setting a base font size of 10px give us easier rem calculations
+         Info: 1rem === 10px, 1.5rem === 15px, 2rem === 20px and so forth
+       */html {
+    
+        font-size: 10px;
+    }
+    
+    body {
+        /* Native font stack https://getbootstrap.com/docs/4.2/content/reboot/#native-font-stack */
+        font-family: -apple-system, BlinkMacSystemFont, 'Segoe UI', 'Roboto',
+            'Helvetica Neue', Arial, sans-serif;
+        font-size: 1.6rem;
+        line-height: 1.5;
+        text-align: center;
+        color: #333;
+        margin: 0;
+    }
```

Comment:
you should add space before and after the comments. When you add linters it will give you errors about this.

Reply:
:red_circle:  Please rephrase your comments in a polite and considerate manner, like a suggestion, instead of a command. This will make it easy for the student to accept and implement them. 
You can rephrase your comment like this :arrow_down: 
**I think** you should add space before and after the comments. When you add linters it will give you errors about this.

Reply:
I think you should add space before and after the comments to enhance the read ability.

Reply:
@bienvenuushindi I have completed the suggestions accordingly.